L1Trigger/Phase2L1ParticleFlow/src/corrector.cc

```diff
@@ -0,0 +1,193 @@
+#include "L1Trigger/Phase2L1ParticleFlow/src/corrector.h"
```

Comment:
move to interface

Reply:
no, as it's not to be used from outside this package